common/memcached/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.6.0 - 2024/10/15
+* version info added to labels
+  * to allow gatekeeper rules based on them
+* old (non-standard) labels removed
```

Comment:
These non-standard labels are used extensively in all helm charts (just search "app: "), so I'm not sure that their removal in shared helm charts would be beneficial and wouldn't introduce too much confusion.

Another problem is that these labels are being used in alerts not only in these dependency charts but also in parent charts, see, e.g., OpenstackManilaMariadbRowLocks or OpenstackKeystoneDatabaseScrapeMissing.

Reply:
Yes, this change needs some clean ups. We have introduced these standard labels a while ago and with this release the phase of this standardization will be done.